### ISSUE TITLE
Crash report 6.2: No Activity found to handle VIEW on http://xxx.wordpress.com

### ISSUE THREAD

**Issue by maxme** [wordpress-mobile/WordPress-Android#4899]
```
Fatal Exception: android.content.ActivityNotFoundException: No Activity found to handle Intent { act=android.intent.action.VIEW dat=http://xxx.wordpress.com }
       at android.app.Instrumentation.checkStartActivityResult(Instrumentation.java:1632)
       at android.app.Instrumentation.execStartActivity(Instrumentation.java:1424)
       at android.app.Activity.startActivityForResult(Activity.java:3390)
       at android.support.v4.app.BaseFragmentActivityJB.startActivityForResult(BaseFragmentActivityJB.java:48)
       at android.support.v4.app.FragmentActivity.startActivityForResult(FragmentActivity.java:77)
       at android.app.Activity.startActivityForResult(Activity.java:3351)
       at android.support.v4.app.FragmentActivity.startActivityForResult(FragmentActivity.java:859)
       at android.app.Activity.startActivity(Activity.java:3587)
       at android.app.Activity.startActivity(Activity.java:3555)
       at org.wordpress.android.ui.ActivityLauncher.viewCurrentSite(ActivityLauncher.java:137)
       at org.wordpress.android.ui.main.MySiteFragment$4.onClick(MySiteFragment.java:173)
       at android.view.View.performClick(View.java:4421)
       at android.view.View$PerformClick.run(View.java:17903)
       at android.os.Handler.handleCallback(Handler.java:730)
       at android.os.Handler.dispatchMessage(Handler.java:92)
       at android.os.Looper.loop(Looper.java:213)
       at android.app.ActivityThread.main(ActivityThread.java:5225)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:525)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:741)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:557)
       at dalvik.system.NativeStart.main(NativeStart.java)
```

**Comment by daniloercoli** [comment]
@hypest  Any thoughts on this? Is this something that has to do with the new "open wpcom links" into the app?

**Comment by hypest** [comment]
@daniloercoli I think it's not related to the URL capturing for at least two reasons:
* it happens even before the 6.2 release that introduced the WPCOM Post URLs interception (see Fabric reports for `ActivityNotFoundException` occurrences)
* The crash happens by trying to open the current site via `ActivityLauncher.viewCurrentSite()` in `MySiteFragment` and the system fails to find an appropriate handler for some reason. Currently, there is no way to go to that screen by intercepting a URL so, most likely it's unrelated.

**Comment by daniloercoli** [comment]
Ok, thanks for the clarification @hypest!

**Comment by maxme** [comment]
I think this might happen when the user doesn't have a web browser installed on his device.

**Comment by maxme** [comment]
Fixed via #4908
